### PR TITLE
Pushgateway doc — added link to node.js client

### PR DESCRIPTION
**Pull request by razonrus**

### CODE REVIEW

File: content/docs/instrumenting/pushing.md [hunk 1]
```diff
@@ -24,3 +24,5 @@ class.
  * For use from Python see [Exporting to a Pushgateway](https://github.com/prometheus/client_python#exporting-to-a-pushgateway).
 
  * For use from Ruby see the [Pushgateway documentation](https://github.com/prometheus/client_ruby#pushgateway).
+ 
+ * For use from Node.js see the [Pushgateway documentation](https://github.com/siimon/prom-client#pushgateway).
```
Comment (brian-brazil) [inline content/docs/instrumenting/pushing.md]: We only link the official clients here, other clients are on the general client library list.

Comment (beorn7) [reply content/docs/instrumenting/pushing.md]: Perhaps we can put an explanation here, along the lines: “The links above are only for official client libraries. Unofficial client libraries may also offer means to push to the Pushgateway.”

Comment (brian-brazil) [reply content/docs/instrumenting/pushing.md]: We could, this is the first time this has come up tbh.

**Comment by beorn7** [comment]
I have created #1437 for the doc clarification.

Closing this one.

**Pull request closed without merging**